web/templates/typeahead_list_item.hbs

```diff
@@ -27,6 +27,13 @@
 {{~#if should_add_guest_user_indicator}}
     <i>({{t 'guest'}})</i>
 {{~/if}}
+{{#if has_pronouns}}
```

Comment:
This looks like `(guest)` will appear before `(she/her)` etc. I don't know if that's desired or not; it might be worth returning to the CZO discussion on pronoun-placement.

Reply:
Yeah I think the other order is preferable.

Reply:
Apologies, while solving the merge conflicts I did not pay attention to the ordering the newly added `(guest)` attribute. I have swapped the positions and removed the non-breaking zero since the spacing issue has also been resolved.

_Please find the updated screenshots on the main comments thread._